.doc_gen/metadata/polly_metadata.yaml

```diff
@@ -154,6 +199,40 @@ polly_StartSpeechSynthesisTask:
                 - python.example_code.polly.StartSpeechSynthesisTask
   services:
     polly: {StartSpeechSynthesisTask}
+polly_DeleteLexicon:
+  title: Delete an &POL; lexicon using an &AWS; SDK
+  title_abbrev: Delete a lexicon
+  synopsis: delete an &POL; lexicon.
+  category:
+  languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/Polly
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - Polly.dotnetv3.DeleteLexiconExample
+  services:
+    polly: {DeleteLexicon}
+polly_SynthesizeSpeechMarks:
+  title: Synthesize speech from text using speech marks with &POL; using an &AWS; SDK
+  title_abbrev: Synthesize speech from text using marks
+  synopsis: synthesize speech from text using marks with &POL;.
+  category:
+  languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/Polly
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - Polly.dotnetv3.SynthesizeSpeechMarksExample
+  services:
+    polly: {SynthesizeSpeech}
```

Comment:
I would put this as a second excerpt under the main SynthesizeSpeech example instead of as a separate example. You can use descriptions between snippet_tags in the excerpt to describe the difference.

Reply:
Good suggestion, thank you. Fixed.